app/src/main/java/org/simple/clinic/splash/SplashScreen.kt

```diff
@@ -21,33 +44,27 @@ class SplashScreen : Fragment() {
   @Inject
   lateinit var router: Router
 
-  private var binding: ScreenSplashBinding? = null
-
-  private val nextButton
-    get() = binding!!.nextButton
-
   override fun onAttach(context: Context) {
     super.onAttach(context)
     context.injector<Injector>().inject(this)
   }
 
-  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-    binding = ScreenSplashBinding.inflate(inflater, container, false)
-    return binding?.root
-  }
+  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+    return ComposeView(requireContext()).apply {
```

Comment:
Have you considered using https://developer.android.com/reference/kotlin/androidx/fragment/compose/package-summary#extension-functions ?

Reply:
We don't have access to that extension since we don't pull the `fragment-compose` dependency. FWIW, it's an easy enough extension to make internally as well. Will consider it for future PRs.